pep-0715.rst

```diff
@@ -0,0 +1,109 @@
+PEP: 715
+Title: Disabling bdist_egg distribution uploads on PyPI
+Author: William Woodruff <william@yossarian.net>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Status: Draft
+Type: Process
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 06-Jun-2023
+Post-History:
+Resolution:
```

Comment:
Just FYI, you can omit headers that don't (yet) have values like `Resolution`. You might need to do this as I'm not sure if anything keys off the presence of the `Resolution` header.

Reply:
Yeah, you can omit them (particularly Resolution; Post-History and Discussions-To need to be updated either right before or right after merging so it might be helpful to have a reminder), though its not required as while they will be displayed, otherwise nothing I'm aware of keys off them being empty vs missing and the linter won't complain.

Reply:
👍